classes/LBRY_Daemon.php

```diff
@@ -69,15 +69,17 @@ public function address_list($page = 1)
 
     /**
      * Returns the available balance of a current LBRY account
-     * https://lbry.tech/api/sdk#account_balance
-     * @param  string   $address    Wallet Address
-     * @return float                Wallet Balance
+     * https://lbry.tech/api/sdk#wallet_balance
+     * @param  string   $address           Wallet Address
+     * @return array   $wallet_balance    Wallet Balance
+     * 
      */
     public function wallet_balance()
     {
-        try {
-            $result = $this->request('account_balance');
-            return $result->result->available;
+        try { // Convert JSON string to an object
+            $result = $this->request('wallet_balance');
+            return $result;
+            $wallet_balance = $result;
```

Comment:
This statement isn't reachable it looks like? should it be above the return statement?

Reply:
I'm assuming this is just updating the call? Where is $wallet_balance used?

Reply:
Yes you are correct not reachable and unneeded, have removed the $wallet_balance variable and also updated the try - catch to throw updated error message. 
Using the $wallet_balance lines 3-4 and 11 on options-page to drill down and access result->available to output the available, unstaked amount in the wallet, prior was showing a much smaller amount (maybe the amount from one address?) than was actually available for use in the wallet.